sdks/java/io/redis/src/test/java/org/apache/beam/sdk/io/redis/RedisIOTest.java

```diff
@@ -89,37 +92,47 @@ public void testGetReadWithKeyPattern() throws Exception {
       pipeline.run();
   }
 
-  @Test
-  public void testGetReadWithNotMatchingKeyPattern() throws Exception {
-    try (EmbeddedRedis embeddedRedis = new EmbeddedRedis()) {
-      insertKV(embeddedRedis);
+  @Test public void testGetReadWithNotMatchingKeyPattern() throws Exception {
```

Comment:
Put `@Test` on a separate line again? I think most tests in Beam if not all put it on a separate line.

Reply:
Sorry, it was the reformating in IntelliJ. I'm cleaning this.